README.md

```diff
@@ -325,6 +325,31 @@ $complete = $open_ai->chat([
 ]);
 ```
 
+For returning JSON responses from OpenAI you need to include the phrase JSON in your requests. 
+
+```php
+$complete = $open_ai->chat([
+    'model' => 'gpt-3.5-turbo-1106',
+    'messages' => 'messages' => [
```

Comment:
`'messages' => 'messages' => [`

should be
`'messages' => [`

Reply:
Thanks for catching the typo.